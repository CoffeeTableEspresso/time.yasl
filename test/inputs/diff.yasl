const time = __require_c__('time')

const first = time.parse('2000-09-07T00:00:00Z')
const second = time.parse('1996-12-15T00:00:00Z')

echo first - second
echo second - first
