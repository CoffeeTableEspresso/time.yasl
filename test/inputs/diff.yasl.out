117676800.000s
-117676800.000s
